src/org/omegat/core/machinetranslators/ApertiumTranslate.java

```diff
@@ -43,9 +51,13 @@
  * @author Didier Briel
  */
 public class ApertiumTranslate extends BaseTranslate {
-    protected static final String GT_URL = "https://www.apertium.org/apy/translate?q=";
+    protected static final String PROPERTY_APERTIUM_SERVER_CUSTOM = "apertium.server.custom";
+    protected static final String PROPERTY_APERTIUM_SERVER_URL = "apertium.server.url";
+    protected static final String PROPERTY_APERTIUM_SERVER_KEY = "apertium.server.key";
+    protected static final String APERTIUM_SERVER_URL = "https://www.apertium.org/apy";
```

Comment:
I would add `_DEFAULT` or something to `APERTIUM_SERVER_URL` and `APERTIUM_SERVER_KEY` to make it clearer what they are for.

Reply:
Agree, fixed in latest commit.